ceph/utils.py

```diff
@@ -44,6 +45,58 @@ def cleanup_ibmc_ceph_nodes(ibm_cred, pattern, timeout=300):
     log.info(f"Done cleaning up nodes with pattern {pattern}")
 
 
+def create_baremetal_ceph_nodes(cluster_conf, inventory):
+    """
+    Creates the nodes with cluster details provided
+    Args:
+        cluster_conf:
+        inventory:
+
+    Returns:
+
+    """
+    log.info("Creating the baremetal ceph nodes")
+    params = dict()
+    ceph_cluster = cluster_conf.get("ceph-cluster")
+    params["cloud-data"] = inventory.get("instance").get("setup")
+    ceph_nodes = dict()
+    with parallel() as p:
+        for node in range(1, 100):
+            node = "node" + str(node)
+            if not ceph_cluster.get(node):
+                break
+            node_dict = ceph_cluster.get(node)
+            node_params = params.copy()
+            node_params["role"] = RolesContainer(node_dict.get("role"))
+            if node_dict.get("no-of-volumes"):
+                node_params["no-of-volumes"] = node_dict.get("no-of-volumes")
+                node_params["size-of-disks"] = node_dict.get("disk-size")
```

Comment:
Based on our earlier discussion, I was of the understanding that we would get this by using `len(node.volumes)`

Reply:
Do we need to know the `size-of-disks`?